core/nodepool/config/config.go

```diff
@@ -37,7 +40,7 @@ type ProvidedConfig struct {
 	cfg.Experimental        `yaml:",inline"`
 	Private                 bool   `yaml:"private,omitempty"`
 	NodePoolName            string `yaml:"name,omitempty"`
-	providedEncryptService  cfg.EncryptService
+	ProvidedEncryptService  cfg.EncryptService
```

Comment:
@mumoshu The `ProvidedEncryptService` from both `.../core/controlplane/config` and `.../core/nodepool/config` are used in function `ConfigFromBytesWithEncryptService ` of `.../core/root/config`.

But I will re-check all my code to see if there are any new member that could be left private 👍